R/palaeorotate.R

```diff
@@ -51,17 +51,17 @@
 #' - Reconstruction files: The "grid" `method` uses reconstruction files to
 #' spatiotemporally
 #' link present-day geographic coordinates and age estimates with a spatial
-#' grid (1&deg; x 1&deg;) rotated to the midpoint of Phanerozoic (0--540 Ma)
+#' grid (~100 km x 100 km) rotated to the midpoint of Phanerozoic (0--540 Ma)
 #' stratigraphic stages (Geological Timescale, 2020). If specific ages of
 #' rotation are required, or fine-scale spatial analyses are being conducted,
 #' use of the "point" `method` might be preferable for the user (particularly
 #' if occurrences are close to plate boundaries). As implemented, points within
 #' the same grid cell will be assigned equivalent palaeocoordinates due to
 #' spatial aggregation. The reconstruction files provide pre-generated
-#' palaeocoordinates for a grid of 1&deg; x 1&deg;, allowing the past
-#' distribution of fossil occurrences to be estimated efficiently. Access to
-#' the reconstruction files and documentation is available via the
-#' [palaeorotate](https://github.com/LewisAJones/palaeorotate) package.
+#' palaeocoordinates for a grid of ~100 km x 100 km, allowing the past
```

Comment:
I think this (and the zenodo documentation) needs to be updated to the hexagonal grid?

Reply:
Done! Thanks for the catch.